libr/bin/bin.c

```diff
@@ -124,6 +124,12 @@ R_API void r_bin_options_init(RBinOptions *opt, int fd, ut64 baseaddr, ut64 load
 	opt->rawstr = rawstr;
 }
 
+R_API void r_bin_arch_options_init(RBinArchOptions *opt, const char *arch, int bits) {
+	memset (opt, 0, sizeof (*opt));
```

Comment:
ok, going to remove this. It actually was overengineered, because initially I was planning to include os/cpu/etc.... but as said, I ended up leaving that for the future.

Reply:
Ah no wait, I actually did this to ensure there is always a default value. See the `opt->arch = arch? arch: R_SYS_ARCH;` and the `opt->bits = bits? bits: 32;`. It's true you could do this with the C syntax, but you'd need to do it in every call site.

Reply:
use R_SYS_BITS then

Reply:
ok